draft-ietf-httpbis-digest-headers.md

```diff
@@ -581,11 +581,7 @@ digest validation failure at the recipient, preventing the application from
 accessing the representation. Such an attack consumes the resources of both
 endpoints. See also {{digest-and-content-location}}.
 
-Digest fields SHOULD always be used over a connection that provides integrity at
-the transport layer that protects HTTP fields.
-
-A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
-signatures.
+Insecure and collision-prone digest-algorithms do not usually fit signature use cases.
```

Comment:
```suggestion
Most signature use cases require transitive trust between parties, and in these cases insecure and collision-prone digest-algorithms ought to be avoided for security purposes.
```

Reply:
Since  [we mention a requirement on adversarial cases, here](https://www.ietf.org/id/draft-ietf-httpbis-digest-headers-07.html#section-6-8) I would reference that..
```suggestion
Since signatures are meant to protect against adversarial use cases, insecure and collision-prone algorithms do not provide adequate guarantees (see {{algorithms}}).
```